test/onnx/verify_onnx.cpp

```diff
@@ -590,6 +590,28 @@ TEST_CASE(if_else_test)
     p.compile(migraphx::ref::target{});
     migraphx::shape s_data{migraphx::shape::float_type, {2, 3}};
     std::vector<float> data = {0.0625, 0.75, -0.0625, 0.125, -0.125, -0.5625};
+    migraphx::shape bool_data{migraphx::shape::bool_type, {1}};
+    bool b_data = false;
+
+    migraphx::parameter_map pp;
+    pp["x"]    = migraphx::argument(s_data, data.data());
+    pp["y"]    = migraphx::argument(s_data, data.data());
+    pp["cond"] = migraphx::argument(bool_data, &b_data);
+
+    auto result = p.eval(pp).back();
+    std::vector<float> result_vector;
+    result.visit([&](auto output) { result_vector.assign(output.begin(), output.end()); });
+
+    std::vector<float> gold = {0.0866565, -0.371067, 0.017719, 0.0250614, 0.0612539, -0.744683};
+    EXPECT(migraphx::verify_range(result_vector, gold));
+}
+
+TEST_CASE(if_else_test_inlined)
```

Comment:
Do we support returning multiple args from MIGraphX program during evaluation ? 
I have vague memory that we don't. 
If we do, we should also add atleast add one of the multi-outputs onnx tests here.

Reply:
> Do we support returning multiple args from MIGraphX program during evaluation ?

Yes we do. For example, bert modles have 2 outputs(which we return).

What we dont support is inlining a subgraph(from `if`) with multiple outputs because we need to have a `make_tuple` operator which requires us to refactor `get_output_alias`.

Reply:
> What we dont support is inlining a subgraph(from if) with multiple outputs because we need to have a make_tuple operator which requires us to refactor get_output_alias.

Ah wait, we can support this by mapping the return to the vector we return from `parse_if`.

Reply:
Yeah I have it running in an test case right now. Going to upload this.

Reply:
Can we add one multi output test in `onnx_test` ?

Reply:
Sure. Adding that now